starter/js/atm.js

```diff
@@ -1,23 +1,72 @@
 //Begin with the document ready function
+$(document).ready(function() {
+  //Checking account deposit function
+  //On click of the depositChecking button
 
-    //Checking account deposit function
+  $("#depositChecking").click(function(event) {
+      console.log(event);
 
-      //On click of the depositChecking button
+      //Get value from the amountChecking input field
+      var depositAmount = parseInt($("#amountChecking").val(), 10);
+      var currentBalance = parseInt($('#checkingBalance').text().replace('$', ''), 10);
```

Comment:
Rather than storing the balances for both the checking and savings accounts in the DOM (in this case, the `#checkingBalance` element) where else could we have stored that data to refer to later?